Library/TeamTalkLib/build/libvpx/CMakeLists.txt

```diff
@@ -19,14 +19,14 @@ if (MSVC)
   endif()
 
   if (TOOLCHAIN_BUILD_EXTERNALPROJECTS)
-    # Favor Cygwin or MSYS2 since CMake may otherwise find a
-    # non-working bash.exe file. GitHub e.g. finds C:/Program
+    # Favor MSYS2 since CMake may otherwise find a non-working
+    # bash.exe file. GitHub e.g. finds C:/Program
     # Files/Git/bin/bash.exe which cannot be used.
-    find_program(BASH_EXE_PATH bash.exe PATHS C:/cygwin64/bin C:/msys64/usr/bin $ENV{PATH} NO_DEFAULT_PATH)
+    find_program(BASH_EXE_PATH bash.exe PATHS C:/msys64/usr/bin $ENV{PATH} NO_DEFAULT_PATH)
```

Comment:
I don't have any plans for changing from Cygwin to msys64 on my development PC. Anyone else prefers msys64 to Cygwin?